tsconfig.json

```diff
@@ -17,10 +17,11 @@
       "iti/utils": [ "../node_modules/intl-tel-input/build/js/utils" ]
     },
     "types": [
-      "node", 
+      "node",
       "jest",
       "@testing-library/jest-dom",
-    ]
+    ],
+    "resolveJsonModule": true,
```

Comment:
Is this used for the fixtures to work?

Reply:
Yes, I took it from [here](https://www.typescriptlang.org/tsconfig#resolveJsonModule)

Reply:
Nice addition 👍